.github/workflows/build-admin-ui.yaml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   dependencies:
     name: Install dependencies
-    runs-on: ubuntu-latest
+    runs-on: ${{ fromJSON(vars.RUNNER) }}
```

Comment:
For my learning sake: Where are we defining `vars.RUNNER`

Reply:
It's in the repo settings
<img width="331" alt="image" src="https://github.com/user-attachments/assets/f1657592-d73b-4e83-982f-5c15317a4a02" />

Reply:
this is neat, I didn't know you could use vars and switch runners based on repos

Reply:
I think I might be missing repo permissions, I am not seeing the screenshot you posted from repo settings, are we talking [about these settings](https://github.com/hashicorp/boundary-ui/settings)?

Reply:
Yes, you'll need admin access which I don't think you have, I just request it for a few hours or a day to change these